### PR TITLE
Feat: add new clickable area variables

### PR DESCRIPTION
**Pull request by marcoambrosini**
Add new variables for clickable areas and reduces size of `--default-clickable-area` 

Part of [#45657](https://github.com/nextcloud/server/issues/45657)

### CODE REVIEW

**Review by artonge — APPROVED** [review]

**Review by susnux — APPROVED** [review]
Maybe keep the large clickable area at 44px, but either way ok for me :)

File: apps/theming/lib/Themes/DefaultTheme.php [hunk 1]
```diff
@@ -172,7 +172,9 @@ public function getCSSVariables(): array {
 			// pill-style button, value is large so big buttons also have correct roundness
 			'--border-radius-pill' => '100px',
 
-			'--default-clickable-area' => '44px',
+			'--default-clickable-area' => '32px',
```
Comment (susnux) [inline apps/theming/lib/Themes/DefaultTheme.php:175]:
```suggestion
			'--default-clickable-area' => '34px',
```

**Review by AndyScherzinger — APPROVED** [review]

**Review by jancborchardt — COMMENTED** [review]
Should clickable-area-large not be 44px, so we don't make elements even bigger?

**Comment by marcoambrosini** [comment]
@jancborchardt sorry I missed your comment, I thought it'd be good to boost it a bit since it would be only one button per app basically. This would be also in line with what google and dropbox do. But we can see how it looks and dial it back before release in case it's too much

**Pull request merged**